tests/test_units_operator.py

```diff
@@ -38,6 +39,7 @@ async def units_operator() -> UnitsOperator:
     units_agent = RoleAgent(container, "test_operator")
     units_role = UnitsOperator(available_markets=[marketconfig])
     units_agent.add_role(units_role)
+    units_agent._role_context.data.update({})
```

Comment:
Well, for some reason, otherwise I can't update it later, because it does not exist.
I think it's because we do not create a world here, where the context data is defined.

Reply:
Now it works...